### PR TITLE
SM64: Verbosify SM64 documentation

### PR DESCRIPTION
**Pull request by wordfcuk**
Docs: verbosify the manual build sections for SM64

## What is this fixing or adding?
Many users have struggled with manually building SM64, or building SM64 at all. This seeks to minimize the amount of Tech Support issues opened with regards to SM64.

## How was this tested?
See [1118719784461279242](https://discord.com/channels/731205301247803413/1118719784461279242)

### CODE REVIEW

**Comment by ThePhar** [comment]
@N00byKing

File: worlds/sm64ex/docs/setup_en.md [hunk 1]
```diff
@@ -37,19 +37,31 @@ After it's done, the Build list should have another entry titled with what you n
...
+*Windows Instructions*
 
-1. Clone `https://github.com/N00byKing/sm64ex` recursively
-2. Enter `sm64ex` and copy your Rom to `baserom.REGION.z64` where `REGION` is either `us` or `jp` respectively.
-3. Compile with `make`. For faster compilation set the parameter `-jn` where `n` is the Number of CPU Cores.
+First, install [MSYS](https://www.msys2.org/) as described on the page. DO NOT INSTALL INTO A FOLDER PATH WITH SPACES.
+
+After launching msys2, and update by entering `pacman -Syuu` in the command prompt. Next, install the relevant dependencies by entering `pacman -S unzip mingw-w64-x86_64-gcc mingw-w64-x86_64-glew mingw-w64-x86_64-SDL2 git make python3 mingw-w64-x86_64-cmake`. SM64EX will link `jsoncpp` dynamic if installed. If not, it will compile and link statically.
+
+After this, obtain the code base by cloning the relevant repository manually via `git clone --recursive https://github.com/N00byKing/sm64ex`. Ready your ROM by copying your legally dumped rom into your sm64ex folder (which is usually located at `msys64/home/user`). The name of the ROM needs to be `baserom.REGION.z64` where `REGION` is either `us` or `jp` respectively.
```
Comment (N00byKing) [inline worlds/sm64ex/docs/setup_en.md:48]: "usually located at" remove that note. Rather not assume where users were before cloning the repo

**Comment by N00byKing** [comment]
Apart from noted, no real objections

File: worlds/sm64ex/docs/setup_en.md (hunk 1 above)
Comment (wordfcuk) [reply worlds/sm64ex/docs/setup_en.md]: Changes have been adressed : )

**Review by Berserker66 — APPROVED** [review]
I didn't read it

**Pull request merged**